### PR TITLE
Filter row

### PR DESCRIPTION
**Pull request by titovmx**

### CODE REVIEW

File: src/core/column-type/filter.row.column.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export class FilterRowColumnModel extends ColumnModel {
+	constructor(model) {
+		super();
+
+		Object.assign(this, model);
+
+		this.key = `$filter.row.${model.key}`;
+		this.type= 'filter-row';
+		this.class = 'filter';
```
Comment (klumba12) [inline src/core/column-type/filter.row.column.js]: setup it as 'control' not 'filter' for now

File: src/core/head/head.view.js [hunk 2]
```diff
@@ -76,5 +108,11 @@ export class HeadView extends View {
 		Log.info('view.head', 'invalidate');
 
 		this.rows = model.view().columns;
+
+		if (model.filter().unit === 'row') {
```
Comment (klumba12) [inline src/core/head/head.view.js]: what happen when user setup filter.unit to default? does it dissapering now?

File: src/core/column-type/column.model.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ export class ColumnModel {
 		this.pin = null;
 		this.origin = 'specific';
 		this.source = 'user';
-		this.class = 'data';
+		this.class = '';
```
Comment (klumba12) [inline src/core/column-type/column.model.js]: Doesnt it break user defined columns?

Comment (titovmx) [reply src/core/column-type/column.model.js]: I didn't see direct problems.
User can define both data and control columns, so in my opinion he has to define class himself when he need.

Comment (klumba12) [reply src/core/column-type/column.model.js]: I think when user create own column he shouldn't think about it class, class is done for internal uses to the most of time

File: src/core/column/column.view.js [hunk 4]
```diff
@@ -46,6 +46,13 @@ export class ColumnView extends View {
 				service.invalidate('column.view', e.changes, PipeUnit.column);
 			}
 		});
+
+		model.filterChanged.watch(e => {
```
Comment (klumba12) [inline src/core/column/column.view.js]: i think this should be in headview itself without calling pipe in other case u use two different approaches: pipe and headview.invalidate

**Review by klumba12 — APPROVED** [review]

**Pull request merged**